### ISSUE TITLE
Claiming Stake Address & Favorites Nothing Showing In Claimed/Favorite Addresses

### ISSUE THREAD

**Issue by FallenStack** [papacarp/pooltool.io#136]
Recently create a new account with one of my stake addresses I wanted to claim after searching up one of the pools I was delegated too and clicked the "Claim address", I used the new account signup option, and send the required ADA to the provided address. The transaction confirmed and the website said it was successful. After checking under my profile, it's show no claimed addresses, additionally when I went to try to re-claim it to see if it worked, it was not allow me saying it was "already claimed", I also tried to favorite the address and it wouldn't show up in my favorite addresses under my profile either.

**Comment by papacarp** [comment]
did this correct itself.  They seem to correct themselves within an hour
historically.  Feel free to dm me on telegram and we can look at your
specific address.

**Comment by jsgonsette** [comment]
+1 for this issue, and waiting one hour didn't change anything.
I have created an account based on my pool reward stake address and paid the 1ADA fee successfully. However when I go to the "profile" section I get this message: 
> No API Keys. You need to claim a pool reward address to get your API key

As my account is already based on the pool reward address, I'm not sure to understand what this means.

In addition, the "Claimed Address" section is empty.
Can someone give a bit more explanation on the procedure to follow to get the API key ?

**Comment by haaptics** [comment]
exact same issue as jgonsette .... has this been addressed or solved?

**Comment by jsgonsette** [comment]
@haaptics Yes it has been resolved by just... waiting a bit longer. I have been told that some scripts were down when I had the problem and it was fixed a bit later.
